ironfish/src/mining/webhooks/webhookNotifier.ts

```diff
@@ -7,6 +7,8 @@ import { createRootLogger, Logger } from '../../logger'
 import { displayIronAmountWithCurrency, ErrorUtils, oreToIron } from '../../utils'
 import { FileUtils } from '../../utils/file'
 
+const BLOCK_EXPLORER_URL = 'https://explorer.ironfish.network'
```

Comment:
Can we open an other PR after this to run these types of things from the config? Remember that some users have custom block explorers and need to customize these settings to point at their custom infrastructure. It's probably best to pass this down into the constructor from the config when the notifiers / pools are created.